specification/servicefabric/resource-manager/Microsoft.ServiceFabric/stable/2021-06-01/cluster.json

```diff
@@ -0,0 +1,1714 @@
+            "type": "string",
+            "description": "The cluster code version."
+          }
+        }
+      },
+      "description": "The list of intermediate cluster code versions for an upgrade or downgrade. Or minimum and maximum upgradable version if no target was given"
+    },
+    "UpgradableVersionsDescription": {
+      "type": "object",
+      "required": [
+        "targetVersion"
+      ],
+      "properties": {
+        "targetVersion": {
+          "type": "string",
+          "description": "The target code version."
+        }
+      }
+    },
+    "UpgradeMode": {
+      "type": "string",
+      "description": "The upgrade mode of the cluster when new Service Fabric runtime version is available.\n\n  - Automatic - The cluster will be automatically upgraded to the latest Service Fabric runtime version as soon as it is available.\n  - Manual - The cluster will not be automatically upgraded to the latest Service Fabric runtime version. The cluster is upgraded by setting the **clusterCodeVersion** property in the cluster resource.\n",
+      "enum": [
+        "Automatic",
+        "Manual"
+      ]
+    },
+    "ErrorModel": {
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorModelError"
+        }
+      },
+      "description": "The structure of the error."
+    },
+    "ErrorModelError": {
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "The error code."
+        },
+        "message": {
+          "type": "string",
+          "description": "The error message."
+        }
+      },
+      "description": "The error details."
+    },
+    "ServiceTypeHealthPolicy": {
+      "type": "object",
+      "description": "Represents the health policy used to evaluate the health of services belonging to a service type.\n",
+      "properties": {
+        "maxPercentUnhealthyServices": {
+          "type": "integer",
+          "description": "The maximum percentage of services allowed to be unhealthy before your application is considered in error.\n",
+          "default": 0,
+          "minimum": 0,
+          "maximum": 100
+        }
+      }
+    },
+    "ServiceTypeHealthPolicyMap": {
+      "type": "object",
+      "description": "Defines a ServiceTypeHealthPolicy per service type name.\n\nThe entries in the map replace the default service type health policy for each specified service type.\nFor example, in an application that contains both a stateless gateway service type and a stateful engine service type, the health policies for the stateless and stateful services can be configured differently.\nWith policy per service type, there's more granular control of the health of the service.\n\nIf no policy is specified for a service type name, the DefaultServiceTypeHealthPolicy is used for evaluation.\n",
+      "additionalProperties": {
+        "$ref": "#/definitions/ServiceTypeHealthPolicy"
+      }
+    },
+    "SystemData": {
```

Comment:
>SystemData [](http://example.com/codeflow?start=5&length=10)

Same comment as in other file. Please refer common definitions
[https://github.com/Azure/azure-rest-api-specs/blob/bf44a3b7e5817ffc57b205a6d63c283af96d222b/specification/common-types/resource-management/v2/types.json](https://github.com/Azure/azure-rest-api-specs/blob/bf44a3b7e5817ffc57b205a6d63c283af96d222b/specification/common-types/resource-management/v2/types.json)